static/app/components/events/eventReplay/replayContent.tsx

```diff
@@ -57,35 +56,43 @@ function ReplayContent({orgSlug, replaySlug, event}: Props) {
     );
   }
 
+  const fullReplayUrl = {
+    pathname: `/organizations/${orgSlug}/replays/${replaySlug}/`,
+    query: {
+      t_main: 'console',
+      f_c_search: undefined,
+      t: initialTimeOffset,
+    },
+  };
+
   return (
     <ReplayContextProvider replay={replay} initialTimeOffset={initialTimeOffset}>
-      <PlayerContainer ref={fullscreenRef} data-test-id="player-container">
+      <PlayerContainer data-test-id="player-container">
         <BadgeContainer>
           <FeatureText>{t('Replays')}</FeatureText>
           <ReplaysFeatureBadge />
         </BadgeContainer>
-        <ReplayView
-          toggleFullscreen={toggleFullscreen}
-          showAddressBar={false}
-          controlBarActions={
-            <Button
-              data-test-id="view-replay-button"
-              to={{
-                pathname: `/organizations/${orgSlug}/replays/${replaySlug}/`,
-                query: {
-                  t_main: 'console',
-                  f_c_search: undefined,
-                  ...(initialTimeOffset ? {t: initialTimeOffset} : {}),
-                },
-              }}
-              priority="primary"
-              size="sm"
-              icon={<IconPlay size="sm" />}
-            >
-              {t('View Full Replay')}
-            </Button>
-          }
-        />
+        <FluidHeight>
+          <CTAOverlayLink aria-label={t('View Full Replay')} to={fullReplayUrl}>
+            <CTAIcon />
+          </CTAOverlayLink>
+
+          <StaticPanel>
+            <ReplayPlayer />
+          </StaticPanel>
+        </FluidHeight>
+
+        <CTAButtonContainer>
+          <Button
+            data-test-id="view-replay-button"
```

Comment:
This is existing code, but `data-test-id` on buttons shouldn't be needed, we should be using `aria-label` selector for all buttons.

Reply:
updated, there are two buttons on the page with that label, and pointing to the same href.